### PR TITLE
fix: windows build issue

### LINKED ISSUE #5610: Build issue in windows
When we run `npm run build` or `npm run build-incremental` in git bash or something like that, it throws below error.

image

### PR DESCRIPTION
**Pull request by ioanatflowcrypt**
This PR fixed build issue in windows system

close #5610 // if this PR closes an issue

----------------------------------

**Tests** _(delete all except exactly one)_:
- Does not need tests (refactor only, docs or internal changes)

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [x] addresses the issue it closes (if any)
- [x] code is readable and understandable
- [x] is accompanied with tests, or tests are not needed
- [x] is free of vulnerabilities
- [x] is documented clearly and usefully, or doesn't need documentation

### CODE REVIEW

**Comment by ioanatflowcrypt** [comment]
@sosnovsky Maybe better to include below tutorial to install rsync in git bash in our readme?

In windows git bash, rsync is not installed by default and we use rsync in build.sh.

> Here's tutorial to install rsync in git bash.
> 
> https://www.chrisborkowski.com/git-bash-here-and-rsync-it/

File: README.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ $ npm install
 $ npm run build
 ```
 
+- For building project on Windows we recommend to use [Cygwin](https://www.cygwin.com/) with rsync package.
```
Comment (sosnovsky) [inline README.md:28]: let's also remove `(currently available for Linux and Mac only)` from line 22, if you were able to successfully build extension on Windows

Comment (ioanmo226) [reply README.md:28]: Sounds good

**Review by sosnovsky — APPROVED** [review]

**Comment by sosnovsky** [comment]
@ioanmo226 looks like CI checks are not performed for non-FlowCrypt contributors, probably it'll be easier if you'll just create another PR with your usual account

**Pull request closed without merging**